app/controllers/user_controller.py

```diff
@@ -4,86 +4,105 @@
 import datetime
 from flask_jwt_extended import *
 
+
 def get_all_users():
     try:
         users = User.query.all()
-        return jsonify({'users': [
+        return jsonify(
```

Comment:
I see a lot of jsonify.

Can we use a helper function for it instead?

To make it more tidy